src/api_routes_http.rs

```diff
@@ -239,6 +240,7 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/delete", web::post().to(delete_post))
           .route("/remove", web::post().to(remove_post))
           .route("/mark_as_read", web::post().to(mark_post_as_read))
+          .route("/mark_many_as_read", web::post().to(mark_posts_as_read))
```

Comment:
I don't know of a better name, but I'd still like to keep this endpoint separate, to preserve the different response type.

Reply:
There is the API v4 stuff, so I don't think it's a massive deal since the API is liable to change anyway.